vault-core/src/test/java/org/apache/jackrabbit/vault/packaging/integration/PackageInstallIT.java

```diff
@@ -376,17 +377,17 @@ public void testExternalHookFailsInInstalledPhase() throws RepositoryException,
         assertNodeExists("/testroot");
     }
 
-    /**
-     * Installs a package with no properties
-     */
-    @Test
     public void testNoProperties() throws RepositoryException, IOException, PackageException {
-        File tmpFile = File.createTempFile("vlttest", "zip");
-        IOUtils.copy(getStream("/test-packages/tmp_no_properties.zip"), FileUtils.openOutputStream(tmpFile));
-        JcrPackage pack = packMgr.upload(tmpFile, true, true, "testpackage", false);
-        assertNotNull(pack);
-
-        pack.install(getDefaultOptions());
+        File tmpFile = File.createTempFile("avlttest", "zip");
```

Comment:
Leveraging https://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html would be even better, as that always cleans up (i.e. no need for try...finally)

Reply:
I can do that, but it makes it harder to diagnose issues because of the now completely random file name.

Reply:
Also, the "try" is still needed so that the output stream gets closed.